crates/moonbuild/src/entry.rs

```diff
@@ -107,6 +110,10 @@ pub fn n2_run_interface(
             });
     };
 
+    if moonbuild_opt.build_graph {
+        vis_build_graph(&state, moonbuild_opt);
```

Comment:
Do we need to print messages to tell the user the location of the outputted dot file?